server/pyproject.toml

```diff
@@ -0,0 +1,22 @@
+[tool.poetry]
+name = "ethereum-indexer-graphql-server"
+version = "0.1.0"
+description = "Graphql server for blockchain indexer"
+authors = ["Your Name <you@example.com>"]
```

Comment:
"RKL <hello@rl.games>"

same in the other one, if I have missed it

Reply:
fixed